meercat/dashboard/views.py

```diff
@@ -469,10 +469,10 @@ def diffCommitData(request):
             except Exception as e:
                 linter_results.append( {'filename': filename, 'results':str(e)} )
         if filename.endswith('.F90'): 
-            output = os.popen('fortran-linter '+str(settings.REPOS_DIR)+'/'+pr.project.name+'/'+filename+' --syntax-only').read()
+            output = os.popen('export PYTHONPATH=${PYTHONPATH}:'+os.path.abspath(str(settings.REPOS_DIR)+'/'+pr.project.name)+' ; cd '+str(settings.REPOS_DIR)+'/'+pr.project.name+' ; . ../meercat/env/bin/activate ; fortran-linter '+str(settings.REPOS_DIR)+'/'+pr.project.name+'/'+filename+' --syntax-only').read()
```

Comment:
This is not specific to this line, but it would be good to run the python code through an autoformatter, e.g., `black`.

Reply:
Ok, I'll give this a try soon.

Reply:
I used black views.py.  I'll use black on other files as changes are made.